clang-tools-extra/docs/clang-tidy/checks/modernize/use-std-print.rst

```diff
@@ -109,7 +109,7 @@ Options
 
   .. code-block:: c++
 
-    std::print("{} {}\n", static_cast<unsigned int>(i), static_cast<int>(u));
+    std::print("{} {}\n", static_cast<int>(u), static_cast<unsigned int>(i));
```

Comment:
The arguments shouldn't be swapped, I think the initial code block above must swap the `%d` and `%u` specifier for this example to make sense.